bundles/org.openhab.binding.bluetooth.hdpowerview/src/main/resources/OH-INF/thing/thing.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluetooth"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Shade Thing Type -->
+	<thing-type id="shade">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="roaming"/>
+			<bridge-type-ref id="bluegiga"/>
+			<bridge-type-ref id="bluez"/>
+		</supported-bridge-type-refs>
+
+		<label>PowerView Shade</label>
+		<description>Hunter Douglas (Luxaflex) PowerView Gen3 Shade</description>
+
+		<channels>
+			<channel id="primary" typeId="primary"/>
+			<channel id="secondary" typeId="secondary"/>
+			<channel id="tilt" typeId="tilt"/>
+			<channel id="battery-level" typeId="system.battery-level"/>
+			<channel id="rssi" typeId="rssi"/>
```

Comment:
The rssi typeid is not declared in this file. This looks like a common percentage, just as a suggestion, (might not fully fit) you might also look at the [system.signal-strength](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-trigger-channel-types) channel

Reply:
Yes it is signal strength

Reply:
This new binding is a child of `org.openhab.binding.bluetooth` and it inherits the `rssi` channel implementation from the parent; therefore the channel definition here in the child simply refers to the [channel-type definition of the parent](https://github.com/openhab/openhab-addons/blob/977105f5ae02f4554f650a87c1999d963c70ff13/bundles/org.openhab.binding.bluetooth/src/main/resources/OH-INF/thing/channels.xml#L7)

Notes: the `rssi` channel value is measured in 'dBm' (a measure of power) .. so:
1. It is is definitely NOT appropriate to use the the `system.signal-strength` channel type.
2. But probably we need to open another PR in `org.openhab.binding.bluetooth` to change its item-type definition (see link above) from `Number` to `Number:Power` => @lsiepel WDYT?

Reply:
As the rssi channel is inherited from the bluetooth binding it is fine for this PR.
A seperate PR changing the channel type for the rssi channel might be usefull, at this moment i cannot asses allt he implications on the dependend bindings.